lib/timeline/Timeline.js

```diff
@@ -83,6 +83,7 @@ function Timeline (container, items, groups, options) {
   this.components = [];
 
   this.body = {
+    getWindow: this.getWindow,
```

Comment:
i should add this function to this.body because i want to get the range in itemSet module and this.body sent to itemSet constructor